client/src/App.js

```diff
@@ -93,30 +144,47 @@ export default class App extends Component {
     // console.log('page:', e.target.name)
     this.setState({
       page: e.target.name
-    })
-
+    });
   }
 
   // When action tiles and navbar are active, remove handlePageChange fn and buttons (Jay)
   render() {
-    if (this.state.page === 'home') {
+    if (this.state.page === "home") {
       return (
         <>
           <SearchAppBar toggleModal={this.toggleModal} />
-          <LandingPage topics={[]} toggleModal={this.toggleModal}/>
-          <Modal modalType={this.state.modalType} isOpen={this.state.isOpen} toggleOpen={this.toggleModal} setLogin={this.setLoginState}/>
-          <button name="action" onClick={(e) => this.handlePageChange(e)}>Go To Action Page</button>
+          <LandingPage topics={[]} toggleModal={this.toggleModal} />
+          <TopicTiles
```

Comment:
This might need to go into the landing page component. Will discuss as team.

Reply:
moved content to the landing page